main.js

```diff
@@ -26,15 +26,37 @@ async function run() {
 
     const fileDiffs = diffSizes(normaliseFingerprint(masterAssets), normaliseFingerprint(prAssets));
 
-    const body = buildOutputText(fileDiffs);
+    const uniqueCommentIdentifier = '_Created by ember-asset-size-action_';
+    const body = `${buildOutputText(fileDiffs)}\n\n${uniqueCommentIdentifier}`;
 
-    try {
-      await octokit.issues.createComment({
+    const updateExistingComment = getInput('update-comments', { required: false });
+    let existingComment = false;
+
+    if (updateExistingComment) {
+      const { data: comments } = await octokit.issues.listComments({
         owner: context.repo.owner,
         repo: context.repo.repo,
         issue_number: pullRequest.number,
-        body,
       });
+      existingComment = comments.find(comment => comment.user.login === 'github-actions[bot]' && comment.body.endsWidth(uniqueCommentIdentifier));
```

Comment:
Ideally we'd have a better way of identifying the existing comment, but this should do for now

Reply:
Coming back to this, we probably don't want to filter comments by the login, since the user may use their own bot to run this action (which may not have the name `github-actions[bot]`)

Reply:
Looking at this, https://api.github.com/users/github-actions%5Bbot%5D
We might be able to check `comment.user.type === 'Bot'` instead

Reply:
I actually think this is the best way to identify the comment 🎉 I added a link back to the project to help people know where it comes from though ;) 

I'll test if this still works before merging this MR though, I don't want to break things just for a link-back 🙈